samples/CloudClipboard/CloudClipboard/CloudClipboard/CloudClipboard.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <LangVersion>preview</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Azure.Identity" Version="1.3.0" />
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.7.0" />
+    <PackageReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="2.1.2" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="1.3.1" />
+    <PackageReference Include="Microsoft.Azure.Storage.Blob" Version="11.2.2" />
```

Comment:
Do we need both storage libraries?

Reply:
Yes, this was a "Track 1 vs Track 2 sample" from https://www.youtube.com/watch?v=Hxne-iM1CtI.  We should leave it so the video still makes sense (and the same feedback for all the remove Track 1 comments).

Reply:
both storage libraries are kept